tests/apollo/test_skvbc_reconfiguration.py

```diff
@@ -68,6 +77,27 @@ def _construct_reconfiguraiton_wedge_status(self):
         reconf_msg.additional_data = bytes()
         return reconf_msg
 
+    def _construct_reconfiguraiton_prune_request(self, latest_pruneble_blocks):
```

Comment:
Nit: change `reconfiguraiton` to `reconfiguration`.

Reply:
done